cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -2,63 +2,57 @@
 import fauxfactory
 import pytest
 
+from cfme.cloud.provider.openstack import OpenStackProvider
 from cfme.fixtures.provider import rhel69_template
 from cfme.fixtures.provider import rhel7_minimal
 from cfme.fixtures.provider import ubuntu16_template
 from cfme.fixtures.provider import win7_template
-from cfme.fixtures.v2v import _form_data
+from cfme.fixtures.v2v_fixtures import get_migrated_vm
+from cfme.fixtures.v2v_fixtures import infra_mapping_default_data
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.markers.env_markers.provider import ONE_PER_VERSION
-from cfme.utils.appliance.implementations.ui import navigator
 from cfme.utils.generators import random_vm_name
 from cfme.utils.log import logger
 from cfme.utils.wait import wait_for
 
+
 pytestmark = [
-    pytest.mark.ignore_stream('5.8'),
     pytest.mark.provider(
-        classes=[RHEVMProvider],
+        classes=[RHEVMProvider, OpenStackProvider],
         selector=ONE_PER_VERSION,
-        required_flags=['v2v'],
-        scope="module"
+        required_flags=["v2v"],
+        scope="module",
     ),
     pytest.mark.provider(
         classes=[VMwareProvider],
         selector=ONE_PER_TYPE,
-        fixture_name='source_provider',
-        required_flags=['v2v'],
-        scope="module"
-    )
+        required_flags=["v2v"],
+        fixture_name="source_provider",
+        scope="module",
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup")
 ]
 
 
-def get_migrated_vm_obj(src_vm_obj, target_provider):
-    """Returns the migrated_vm obj from target_provider."""
-    collection = target_provider.appliance.provider_based_collection(target_provider)
-    migrated_vm = collection.instantiate(src_vm_obj.name, target_provider)
-    return migrated_vm
-
-
-@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+@pytest.mark.parametrize('mapping_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
     indirect=True)
 @pytest.mark.parametrize('power_state', ['RUNNING', 'STOPPED'])
-def test_single_vm_migration_power_state_tags_retirement(request, appliance, v2v_providers,
-                                    host_creds, conversion_tags,
-                                    form_data_vm_obj_single_datastore,
-                                    power_state):
+def test_single_vm_migration_power_state_tags_retirement(request, appliance, provider,
+                                                         mapping_data_vm_obj_single_datastore,
+                                                         power_state):
     """
     Polarion:
-        assignee: kkulkarn
+        assignee: sshveta
         casecomponent: V2V
         initialEstimate: 1/2h
         subcomponent: RHV
```

Comment:
@sshveta in my tests I removed `subcomponent: RHV` since we are using such tests for OSP as well. I'm not sure if we can add two subcomponent to single test but if not, better to remove it from here and we can add that later in polarion directly when this conversion finishes, please apply to other tests as well if you are considering it.